docs/book/src/component-config-tutorial/testdata/project/config/rbac/metrics_role.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     app.kubernetes.io/name: project
     app.kubernetes.io/managed-by: kustomize
-  name: proxy-role
+  name: metrics-role
```

Comment:
This clusterrole was required for the kube-rbac-proxy, and is not required anymore.